controllers/group_totals_test.go

```diff
@@ -70,9 +89,10 @@ func (q *QueryTotalsSuite) TestInstaCountAllHandler(c *C) {
 
 	counts := make(map[string]int)
 	err := json.NewDecoder(w.Body).Decode(&counts)
+
 	util.CheckErr(err)
 
-	c.Assert(counts["patients"], Equals, 5)
-	c.Assert(counts["conditions"], Equals, 10)
-	c.Assert(counts["encounters"], Equals, 12)
+	c.Assert(counts["patients"], Equals, 1)
+	c.Assert(counts["conditions"], Equals, 1)
+	c.Assert(counts["encounters"], Equals, 1)
```

Comment:
This test seems a little too trivial to really give me confidence that bugs in the code would be detected.  I know you've tested this manually, so we know it works.  Could we at least put in a TODO so that we remember to come back and write some better tests (maybe when we have some better test fixtures and helpers)?